source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -0,0 +1,2163 @@
+{
```

Comment:
<p>These column names look like they are the first line of the data. Do we want to tell it the names when loading? Or even writing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>